addons/website/static/src/xml/website.background.video.xml

```diff
@@ -8,6 +8,7 @@
                 </div>
             </div>
             <iframe t-att-id="iframeID"
+                    loading="lazy"
```

Comment:
Now that you modified a website file, you should add the website module in the commit title :wink: